acceptance/tests/facts/windows.rb

```diff
@@ -88,8 +88,13 @@
   end
 
   step "Ensure the identity fact resolves as expected"
+  # Regular expression to validate the username from facter in the form of '<domain>\<username>'
+  # Reference - https://msdn.microsoft.com/en-us/library/bb726984.aspx
+  # - The domain name can be any character or empty
+  # - Must contain a backslash between the domain and username
+  # - Username must be at least one character and not contain the following charaters; " / \ [ ] : ; | = , + * ? < >
   expected_identity = {
-                        'identity.user'       => /.*\\cyg_server/,
+                        'identity.user'       => /.*\\[^\\\/\"\[\]:|<>+=;,?*@]+$/,
```

Comment:
My bigger point was that `.*` is too lenient - sorry, I probably should have been more clear about that.  

You also match strings like:

* `foo\bar\baz` 
* `\\foo\\bar`
* `123!<>!@#$%^\\foo`

At least before we at least had `cyg_server` in there, so the `.*`, while it could accept junk, wasn't as bad.  Now the regex accepts a lot more stuff.